tests/js/spec/views/dashboardsV2/widgetBuilder/buildSteps/visualizationStep.spec.tsx

```diff
@@ -140,40 +140,4 @@ describe('VisualizationStep', function () {
 
     await screen.findByText(/we've automatically adjusted your results/i);
   });
-
-  it('displays a deprecated tag next to the top 5 display option', async function () {
```

Comment:
Deleted these tests because they no longer apply if we remove the top 5 display type.